release-team/release-team-removal.md

```diff
@@ -0,0 +1,52 @@
+The following outline is not meant to be definitive or punitive but rather outlines general guidance for leads on the process.  
+## Shadow Removal Guideline
+
+Reasons for removal could be including but not limited to:
+- Failure to attend shadow orientation hosted by the EA (we host 2-3 sessions in different timezones)
+- Failure to attend shadow orientation hosted by the role lead (if lead doesn't offer timezone-appropriate meeting, shadow should communicate to lead about obtaining and watching the recording)
+- Failure to attend 3 consecutive release meetings without communication to the role lead
+- Failure to perform shadow tasks in due time, after 2-3 discussions with the role lead and the EA about performance
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on the team's responsibility
+- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
+
+Note that communications ahead of time to inform the leads or find alternative arrangements are greatly preferred in these circumstances.
+### Process
+It is important to be kind and patient while trying to resolve the situation.
+- Upon a pattern of inactivity, role lead will notify EA and have initial discussion with shadows about performance
+- If the shadow is still not contributing to the team, EA and lead will remove the shadow from the team, and find a replacement if appropriate. EA will be responsible for finding backfill.
+
+### Consequences
+- If the current release is the reason the shadow has Kubernetes membership, membership should be revoked
+- Removal of name in release-team document
+- Shadow will not be able to return to the following release. However, they are welcome to apply to releases after that
+
+## Role Lead Removal Guideline
+A role lead has been selected by your previous team lead to supervise, perform the duties of your team 
+and mentor your shadows to be future leads. Failure to perform lead duty can have a lasting impact on the release team.
+Reasons for removal could be including but not limited to:
+- Failure to attend or send appropriate representation to 2 consecutive release meetings
+- Failure to perform tasks expected of a role lead
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on team
+- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
+
+### Process
+- SIGs outside of SIG-release, shadows, leads, or contributors can express concern to release lead or EA.
+- Release lead and EA will have initial discussion with role lead
+- If the situation is not improved, the release lead and EA will have another discussion with role lead and begin to look for a replacement.
```

Comment:
```suggestion
- If the situation does not improve within a week after the first discussion, the Release Lead and the EA will have another discussion with the Role Lead and begin to look for a replacement.
```

within a week?

Reply:
it might take more than a week to get signal. this is also more general guideline